var/spack/repos/builtin/packages/elk/package.py

```diff
@@ -3,130 +3,184 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+
+# TODO: respect the spack-profided compiler flags if any
+
+# TODO: add MKL and BLIS support
+# Attention! MKL and BLIS are NOT JUST drop-in replacements
+# for blas/lapack!
+
+# TODO: Check that spack-provided  compiler flags
+# are respected
+
+# Attention!
+# Dynamic linking *appears* to work fine, but
+# Elk is expected to be linked statically.
+
+# TODO: check if it is possible/reasonable to disable OpenMP
+# It doesn't appear to be this way, but check anyway
+
+
 from spack import *
 
 
 class Elk(MakefilePackage):
-    """An all-electron full-potential linearised augmented-plane wave
-    (FP-LAPW) code with many advanced features."""
-
-    homepage = 'http://elk.sourceforge.net/'
-    url      = 'https://sourceforge.net/projects/elk/files/elk-3.3.17.tgz'
-
-    version('3.3.17', sha256='c9b87ae4ef367ed43afc2d43eb961745668e40670995e8e24c13db41b7e85d73')
-
-    # Elk provides these libraries, but allows you to specify your own
-    variant('blas',   default=True,
-            description='Build with custom BLAS library')
-    variant('lapack', default=True,
-            description='Build with custom LAPACK library')
-    variant('fft',    default=True,
-            description='Build with custom FFT library')
-
-    # Elk does not provide these libraries, but allows you to use them
-    variant('mpi',    default=True,
-            description='Enable MPI parallelism')
-    variant('openmp', default=True,
-            description='Enable OpenMP support')
-    variant('libxc',  default=True,
-            description='Link to Libxc functional library')
-
-    depends_on('blas',   when='+blas')
-    depends_on('lapack', when='+lapack')
-    depends_on('fftw',   when='+fft')
-    depends_on('mpi@2:', when='+mpi')
-    depends_on('libxc',  when='+libxc')
-
-    # Cannot be built in parallel
+    """An all-electron full-potential linearised augmented-plane wave (LAPW)
+    code with many advanced features. Written originally at
+    Karl-Franzens-Universität Graz as a milestone of the EXCITING EU Research
```

Comment:
The "invalid syntax" that the CentOS 6 tests catch is this ä. CentOS 6 runs Python 2, which does not support unicode characters.